### PR TITLE
186547814 update validators api

### PR DESCRIPTION
**Pull request by maciejkocylapc**
#### What's this PR do?
- set random order of the results by score
- remove inactive validators from default query (make it optional)
- add is_active field to API response
- update docs

#### How should this be manually tested?
- check if the docs are updated
- run tests
- try to send requests to /api/v1/validators/:network with various params and see if they match the documentation description
- make sure there are no inactive validators by default

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/186547814)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?

### CODE REVIEW

**Review by kbiala — APPROVED** [review]

File: app/queries/validator_query.rb [hunk 1]
```diff
@@ -21,7 +33,7 @@ def call(network: "mainnet", sort_order: "score", limit: 9999, page: 1, random_s
     scope = set_ordering(scope, sort_order, random_seed_val)
     scope = set_pagination(scope, page, limit)
 
-    @api ? scope : scope.scorable
+    query_params[:active_only] ? scope.scorable : scope
```
Comment (kbiala) [inline app/queries/validator_query.rb:36]: This condition is incorrect. All strings, even those with "false" values, are truthy, so this condition will always return active validators.
Please add this scenario to tests - only the opposite scanario is tested.

File: app/views/public/api_documentation.html.erb [hunk 2]
```diff
@@ -118,8 +120,8 @@
     "skipped_slot_percent":"0.5155",
     "ping_time":"205.703",
     "url":"<%= @api_path %>/validators/testnet/12345j84ecQEKUvQ7gYMKxKwKF6PbYXxXxxXXXXXxXXX.json",
-    "stake_pools_list":["Jito", "Marinade"]
-
+    "stake_pools_list":["Jito", "Marinade"],
+    "is_active": true
```
Comment (kbiala) [inline app/views/public/api_documentation.html.erb:124]: I can't see "is_active" attribute in response, no matter which request params I set.

**Review by kbiala — CHANGES_REQUESTED** [review]

File: test/queries/validator_query_test.rb [hunk 3]
```diff
@@ -254,4 +237,30 @@ class ValidatorQueryTest < ActiveSupport::TestCase
 
     assert_equal validator, result.first
   end
+
+  test "#call returns only active validators if active_only is true" do
```
Comment (kbiala) [inline test/queries/validator_query_test.rb:241]: Please also test the result if active_only is empty.

**Review by kbiala — APPROVED** [review]
